pkgs/development/compilers/gcc/default.nix

```
@@ -150,6 +151,7 @@ let
       fetchpatch
       fetchurl
       flex
+      fixDarwinDylibDuplicateLcRpaths
```

```suggestion
      fixDarwinDylibDuplicateLcRpaths
      flex
```

(nit: alphabetically sorted)